### PR TITLE
fix: update volcano admission yaml

### PR DESCRIPTION
**Pull request by youhonglian**
remove "admission-conf" args

### CODE REVIEW

File: installer/volcano-development.yaml [hunk 1]
```diff
@@ -136,7 +136,6 @@ spec:
             - --tls-cert-file=/admission.local.config/certificates/tls.crt
             - --tls-private-key-file=/admission.local.config/certificates/tls.key
             - --ca-cert-file=/admission.local.config/certificates/ca.crt
-            - --admission-conf=/admission.local.config/configmap/volcano-admission.conf
```
Comment (shinytang6) [inline installer/volcano-development.yaml:139]: This configuration is useful and should not be deleted :)

**Review by shinytang6 — CHANGES_REQUESTED** [review]

File: installer/volcano-development.yaml (hunk 1 above)
Comment (Thor-wl) [reply installer/volcano-development.yaml:139]: Why delete admission-conf?

Comment (hwdef) [reply installer/volcano-development.yaml:139]: image

I encountered this error.

Comment (shinytang6) [reply installer/volcano-development.yaml:139]: same issue with https://github.com/volcano-sh/volcano/issues/1663, latest image should be updated.
/cc @Thor-wl

Comment (Thor-wl) [reply installer/volcano-development.yaml:139]: > same issue with #1663, latest image should be updated.
> /cc @Thor-wl

Got it. It seems to be necessary to update recently.

Comment (youhonglian) [reply installer/volcano-development.yaml:139]: > Why delete admission-conf? BTW, please add self signature when push the commit.

Thanks for reminding, I added my signature.
I also encountered this error. so i  updated YAML, is right? :)

Comment (Thor-wl) [reply installer/volcano-development.yaml:139]: Well, thanks for the contribution. But the root reason is not here. I'll fix it.

Comment (youhonglian) [reply installer/volcano-development.yaml:139]: thanks

**Pull request closed without merging**